var/spack/repos/builtin/packages/laghos/package.py

```diff
@@ -21,6 +21,7 @@ class Laghos(MakefilePackage):
     maintainers = ['v-dobrev', 'tzanio', 'vladotomov']
 
     version('develop', branch='master')
+    version('3.1', commit='95f2b9d748146d94ab70086329a6c6f814767379')
```

Comment:
@tzanio, @vladotomov, we should probably tag this commit with `v3.1` in the laghos repo. Adding this version was necessary since v3.0 does not work with mfem v4.2.

Reply:
I am OK with that

Reply:
Done.